Packs/SplunkPy/Integrations/SplunkPy/README.md

```diff
@@ -79,7 +79,7 @@ where the **$IDENTITY_VALUE** is replaced with the **user** and **src_user** fro
 #### How to configure
 1. Configure the integration to fetch incidents.
 2. *Enrichment Types*: Select the enrichment types you want to enrich each fetched notable with. If none are selected, the integration will fetch notables as usual (without enrichment).
-3. *Fetch events query*: The query for fetching events. The default query is for fetching notable events. You can edit this query to fetch other types of events. Note that to fetch notable events, make sure the query uses the \`notable\` macro.  
+3. *Fetch events query*: The query for fetching events. The default query is for fetching notable events. You can edit this query to fetch other types of events. Note that to fetch notable events, make sure the query uses the \`notable\` macro and the results are sorted by time (for example: add `| sort 0 _time` to the query).  
```

Comment:
Please replace the bolded text (I am not able to submit my changes for some reason.)
 ....  \`notable\` macro ** and the results are sorted by time**. See the default query as an example.